kb-importer/pom.xml

```diff
@@ -72,92 +85,36 @@
 			<version>${project.version}</version>
 			<scope>runtime</scope>
 		</dependency>
-		<dependency>
```

Comment:
This is not required any more?

Reply:
It is already present in the parent pom.xml